tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+
+Now you are ready to the the machine-processable Bioshemas version of your new draft profile.
+
+* Go to [GOweb repository](https://github.com/BioSchemas/bioschemas-goweb) and get the executable version corresponding to your Operational System. 
+* Run GOWeb using the CSV file you created on the previous section [Publish the Bioschemas fields to the Web](#23-publish-the-bioschemas-fields-to-the-web)
+* A YAML file will be created, you will use that one on the next section [Update the website](#4-update-the-website)
+
+## 4. Update the website
+
+You are now ready to publish the new draft version on the Bioschemas website. 
+
+* If you have not done so yet, clone the [Bioschemas website repository](https://github.com/BioSchemas/bioschemas.github.io) so you have a local copy
+* Open your local copy in your preferred editor
+* Go to `_profiles/<your profile>`, in this case `_profiles/ScholarlyArticle`
+* Create a copy of the latest profile, in our case it is 0.2-DRAFT-2020_12_03.html
+* Rename the copy so it reflects the new draft version, in our case it would be 0.3-DRAFT.html (having the dates as part of the draft name is no longer needed/desired)
+* On the previous version file, in our case 0.2-DRAFT-2020_12_03.html
+  * Remove the redirect_from version
+
+```
+redirect_from:
+- "devSpecs/ScholarlyArticle/specification"
+- "devSpecs/ScholarlyArticle/specification/"
+- "/devSpecs/ScholarlyArticle/"
+- "/specifications/drafts/ScholarlyArticle"
+- "/profiles/ScholarlyArticle/"
+```
+  
+* Now on the new draft version file
+  * Update the previous version, in our case it would be ```previous_version: 0.2-DRAFT-2020_12_03```
+  * Update the ```cross_walk_url``` to point to the spreadsheet on the GDrive
+  * If you changed the parent type (very rarely it happens), you need to update the section ```parent_type```
+
+  Now you are ready to update the part corresponding to the spreadsheet, go to the ```spec_info``` section, it start with the lines
+  
+```
+  # spec_info content generated using GOWeb
+  # DO NOT MANUALLY EDIT THE CONTENT
+```
+
+* Keep the two first lines only, i.e., remove from the line ```spec_info``` all the way to a line with three dashes ```---```, which is right before the line ```<!DOCTYPE HTML>```
+* Add all the lines comming from the YAML file that you got from GO Web
+* Save the file!
+
+You are now ready to update the profile version
+* Go to the folder _data and locate the file ```profile_versions.yaml```
+* Find there the profile you are updating, in this case ScholarlyArticle, and update the version, in this case it would be version 0.3-DRAFT
+
+```
+ScholarlyArticle:
+    name: "ScholarlyArticle"
+    latest_publication: "0.3-DRAFT"
+    latest_release:
+    status: "active"
+```
+
+## 5. Get an overview on your local copy
+
+Now you are ready to run the website locally, open a terminal and run ```jekyll serve``` (how to get Jekyll running is out of the scope of this tutorial).
```

Comment:
Point to the GitHub help pages for this.

Reply:
done